x/superfluid/keeper/migrate.go

```diff
@@ -332,10 +345,5 @@ func (k Keeper) validateSharesToMigrateUnlockAndExitBalancerPool(ctx sdk.Context
 		return sdk.Coins{}, err
 	}
 
-	// Defense in depth, ensuring we are returning exactly two coins.
-	if len(exitCoins) != 2 {
```

Comment:
not necessarily! "Force Unlock" handles both superfluid locks and non superfluid locks!

Reply:
But changed to have this check back conditionally regardless

Reply:
I'm slightly confused why we have this method in the superfluid module then?

Reply:
I think we can make it to be in the lockup keeper, just that more refactoring needs to be done since we also have migration code depending on this method (happy to address this in future code clean up or furture PRs)

Reply:
@ Roman's comment, wouldn't something that modifies both sf and non sf locks belong in sf? SF module has both SF keepers and lockup keepers, while lockup module does not have access to SF keepers.

Reply:
Makes sense. I'm good with the current approach